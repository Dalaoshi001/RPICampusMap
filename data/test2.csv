"c1","book1"
"c2","book1"
"c3","book1"
"c3","book2"
"c4","book2"
"c4","book3"
"c5","book1"
"c5","book3"