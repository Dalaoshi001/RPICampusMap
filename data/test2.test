r
81
22
r
-1
-2
q