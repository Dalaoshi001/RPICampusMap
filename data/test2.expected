First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from RPI Ambulance to VCC:
	Walk North to (Intersection 99)
	Walk NorthWest to (Cogswell Laboratory)
	Walk North to (Intersection 136)
	Walk NorthWest to (Folsom Library)
	Walk East to (VCC)
Total distance: 417.368 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [-1]
Unknown building: [-2]
